docs/setup.md

```diff
@@ -0,0 +1,21 @@
+# Installation
+
+SARXarray can be installed from PyPI:
+
+```sh
+pip install sarxarray
+```
+
+or from the source:
+
+```sh
+git clone git@github.com:MotionbyLearning/sarxarray.git
+cd sarxarray
+pip install .
+```
+
+Note that Python version `>=3.10` is required for SARXarray.
+
+## Tips
+
+We strongly recommend installing separately from your default Python envrionment. E.g. you can use enviroment manager e.g. [conda](https://docs.conda.io/projects/conda/en/latest/user-guide/install/download.html) to create separate environment.
```

Comment:
I suggest refering users to mamba instead of conda https://mamba.readthedocs.io/en/latest/mamba-installation.html